linera-views/src/scylla_db.rs

```diff
@@ -184,7 +184,10 @@ impl ReadableKeyValueStore<ScyllaDbContextError> for ScyllaDbStoreInternal {
 
 #[async_trait]
 impl DirectWritableKeyValueStore<ScyllaDbContextError> for ScyllaDbStoreInternal {
-    const MAX_BATCH_SIZE: usize = usize::MAX;
+    // The constant 14000 is an empirical constant that was found to be necessary
+    // to make the ScyllaDb system work. We have not been able to find this or
+    // a similar constant in the source code or the documentation.
+    const MAX_BATCH_SIZE: usize = 14000;
```

Comment:
Intuitively, this is a lot of keys already for a single atomic transaction on a distributed KV store. Perhaps it would be wise to take a big safety margin (maybe 2000 ?).

Reply:
The error I obtained while running about the limit of 14796 is
`ScyllaDbQueryError(DbError(WriteFailure { consistency: LocalQuorum, received: 0, required: 1, numfailures: 1, write_type: Simple }, "Operation failed for kv.table_tzhfxlsj8so3ikaof3lf - received 0 responses and 1 failures from 1 CL=LOCAL_QUORUM."))`

Reply:
This seems like undocumented behavior. I would take a much larger safety margin here. It's not a breaking change to increase the value later.